### PR TITLE
Created an api local reset

### PR DESCRIPTION
**Pull request by bsm5**
Sets a station's status to disabled in the cached station object.

### CODE REVIEW

File: routes/api.js [hunk 1]
```diff
@@ -81,5 +82,50 @@ router.post('/station-heartbeat', function(req, res) {
...
+    db.getStation(sid).then(results => {
+      if(results){
+        if(results['registered'] && cert.subject.CN === results['certCN']){
+          if(sCache.updateMachine(sid, 'disabled')){
+           db.logEvent('station-reset', 'Station: ' + sid + ' reset button activated').then(function(){
+             res.sendStatus(200);
+           });
+          }else{
+            db.logEvent('station-reset', 'Station: ' + sid + ' cache error').then(function(){
+              res.sendStatus(403);
```
Comment (mongolsamurai) [inline routes/api.js]: This block looks like it represents an internal error in our software, not a problem with the client. If so, it should be returning a `500` error.

File: routes/api.js [hunk 2]
```diff
@@ -81,5 +82,50 @@ router.post('/station-heartbeat', function(req, res) {
...
+          });
+        }
+      }else{
+        db.logEvent('station-reset', 'Station: ' + sid + ' not found').then(function() {
+        res.sendStatus(403);
+        });
+      }
+    }).catch(err => {
+      db.logEvent('station-reset error', err).then(function() {
+      res.sendStatus(403);
```
Comment (mongolsamurai) [inline routes/api.js]: Also looks like a `500` error to me

File: routes/api.js [hunk 3]
```diff
@@ -81,5 +82,50 @@ router.post('/station-heartbeat', function(req, res) {
...
+      });
+    });
+  }else{
+    res.sendStatus(403);
```
Comment (mongolsamurai) [inline routes/api.js]: This looks to me like it runs when the request is malformed, which should be a `400` error rather than a `403`

**Comment by bsm5** [comment]
@mongolsamurai All requested changes should be fixed now.

**Pull request merged**